bank_accounts_report/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+	:target: http://www.gnu.org/licenses/agpl
+	:alt: License: AGPL-3
+
+====================
+Bank Accounts Report
+====================
+
+Choose which bank accounts will be displayed and payment message on invoice and sale order reports.
+
+
+Installation
+============
+
+Just install.
+
+
+Configuration
```

Comment:
@Jesarregui la configuración la entendéis correctamente tal y como está indicado en el README? @luis-ron @Jaimermaccione @HaraldPanten

Reply:
README might be improved, yes.

I wouldn't explain all the possibilities, but something like this one would be useful:

"User can set these fields in the payment mode configuration":

- Apply in sale orders (if user wants to see the payment mode details in sales orders).
- Partner Account Origin (company, partner)
- ...

Reply:
@luis-ron @Jaimermaccione @Jesarregui ¿podéis realizar el README desde 0 para que la configuración, etc. esté mejor explicado?
Gracias 😄

Reply:
Okey! How do you see it like this?

**Configuration**
**1. Access Payment Modes**
To configure the payment modes that will influence how bank accounts and payment messages are displayed:

- Navigate to **Accounting > Configuration > Payment Modes.**

**2. Configure Each Payment Mode**
For each payment mode, adjust the following options according to your needs on **Bank Account on Reports** section:

- **Account Source:** Choose **Company** to display bank accounts associated with the company on reports. Choose "Partner" to display bank accounts from the partner based on mandates or direct bank accounts.
  - If **Company** is selected, you can opt to **Use Invoice Account**, which will display the account set on the invoice. If this option is unchecked, you can manually select multiple accounts from your company to appear.
  - If **Partner** is selected, decide whether the account will be taken from an existing mandate or simply from the last active bank account of the partner.
- **Apply to Sale Orders:** Check this option if you want the settings to apply to sales orders as well.
- **Report Text:** Enter a custom message that will appear on sales and invoice reports.
- **Payment Message:** If you want a payment message displayed on invoices or sales orders, check the 'Show Payment Message in Invoices' and/or 'Show Payment Message in Sales' checkbox and complete the 'Payment Message' field. Invoices or Sales Orders will show 'Payment Message' + Invoice/Sales name.

**3. Visibility of the Bank Account Number**
Configure how you would like bank account numbers to be displayed on reports:

**How Show Bank Account:** Choose between showing the full number, only the first characters, or only the last characters. Adjust the number of characters to display accordingly.

- **Full:** The entire bank account number is displayed.
- **First n chars:** Only the first specified number of characters of the bank account number will be shown, with the rest being masked.
- **Last n chars:** Only the last specified number of characters of the bank account number will be shown, with the beginning of the number being masked.

**Usage**
**Creating Invoices or Sales Orders**
Once the payment modes are configured, you can proceed to create invoices or sales orders that use these payment modes.

**Verification in Reports**

1. **Create an Invoice/Sales Order:** Ensure to select the previously configured payment mode.
2. **Review the Generated Report:** When generating the report for the invoice or sales order, check the end of the document to see the bank accounts listed as per your configuration. Additionally, any configured payment message should appear as set.

On the other hand, the Spanish translation where I have tested it is confusing; 'partner' is translated as 'empresa' and 'company' as 'compañía'. It might be better to translate 'partner' as 'contact' instead.

Reply:
@Jesarregui @luis-ron como lo veis que ese sea el readme?

Reply:
It's much more than fine. Good job! 👏 

@ValentinVinagre It's enough.

Reply:
@manuelregidor puedes realizar el cambio en el README.rst que indica @Jaimermaccione ? merci 😄